props/saber_BC_buttons.h

```diff
@@ -1267,28 +1288,43 @@ struct BCVolumeMode : public SPEC::SteppedMode {
       if (current_volume_ <= min_volume_) {
         current_volume_ = min_volume_;
         QuickMinVolume();
-      } else {
+      } else if (millis() - trigger_time_ > 100 && !sound_library_.busy()) {
         mode::getSL<SPEC>()->SayVolumeDown();
+        rangeState_ = RangeState::None;
+        trigger_time_ = millis();
       }
       dynamic_mixer.set_volume(current_volume_);
     }
   }
 
   void QuickMaxVolume() {
     dynamic_mixer.set_volume(max_volume_);
+    // Always play immediately when Max reached
```

Comment:
If we're always playing immediately, then why not just call fadeout() unconditionally?
Why check rangeState_ ?

Reply:
We're not always playing immediately. Only max/min. See longer reply above.

Reply:
If we're not always playing immediately, then maybe the comment shouldn't say "Always play immediately" ?